api/src/authkeys/initJWTKeys.ts

```diff
@@ -3,9 +3,12 @@ import {
   COUCHDB_INTERNAL_URL,
   KEY_SERVICE,
   LOCAL_COUCHDB_AUTH,
+  RUNNING_UNDER_TEST,
 } from '../buildconfig';
 
 export async function initialiseJWTKey(): Promise<void> {
+  // don't try to do this if we're testing
+  if (RUNNING_UNDER_TEST) return;
```

Comment:
Might be good to log this skip in case someone misconfigures the API and is confused as to why keys are not being setup appropriately. Do we have logging with levels configured? This could be a debug log.

Reply:
This isn't a config setting, should only be true when tests are running. I think it's safe.

Reply:
Okay